api/pkg/s3/listbuckets.go

```diff
@@ -67,7 +67,7 @@ func (s *APIService) ListBuckets(request *restful.Request, response *restful.Res
 	}
 
 	ctx := common.InitCtxWithAuthInfo(request)
-	rsp, err := s.s3Client.ListBuckets(ctx, &s3.BaseRequest{})
+	rsp, err := s.s3Client.ListBuckets(ctx, &s3.ListBucketsRequest{})
```

Comment:
Why new Request format?

Reply:
because BaseRequest{} is used by few other functions